From: uazo <uazo@users.noreply.github.com>
Date: Mon, 1 May 2023 12:08:46 +0000
Subject: v113 temp fix build

CalyxOS Port Note: We only need the safe_browsing flatbuffers build fix,
at least for now. (...also, we are v116 as of this writing.)

Change-Id: Iddaf7df7ed8cc40fc402473fa4025b8b66bafb39
---
 chrome/browser/safe_browsing/BUILD.gn | 1 +
 1 file changed, 1 insertion(+)

diff --git a/chrome/browser/safe_browsing/BUILD.gn b/chrome/browser/safe_browsing/BUILD.gn
--- a/chrome/browser/safe_browsing/BUILD.gn
+++ b/chrome/browser/safe_browsing/BUILD.gn
@@ -210,6 +210,7 @@ static_library("safe_browsing") {
       "//components/version_info",
       "//content/public/browser",
       "//services/preferences/public/mojom:mojom",
+      "//third_party/flatbuffers:flatbuffers",
     ]
     if (safe_browsing_mode == 1) {
       # "Safe Browsing Full" files in addition to the "basic" ones to use for
-- 

